023_agile_essential.storypoints.md

```diff
@@ -6,11 +6,11 @@ Improve your estimations and release planning.
 
 ## Description
 
-Story points are estimating the implementation complexity of a story, rather than the required time. Estimating in story points will improve the accuracy of your release plans, decrease the estimation effort for your teams and the variation in estimates due to external changes.
+Story points are estimating the implementation complexity of a story, rather than the required time. Estimating in Story Points will improve the accuracy of your release plans, decrease the estimation effort for your teams and the variation in estimates due to external changes.
```

Comment:
Würde dann Story Points auch immer groß schreiben. Hier am Anfang erst klein, danach groß und im in Zeile 11 wieder klein.